lib/govuk_navigation_helpers/taxon_breadcrumbs.rb

```diff
@@ -5,11 +5,18 @@ def initialize(content_item)
     end
 
     def breadcrumbs
-      ordered_parents = all_parents.map do |parent|
-        { title: parent.title, url: parent.base_path }
+      ordered_parents = all_parents.map.with_index do |parent, index|
+        {
+          title: parent.title,
+          url: parent.base_path,
+          is_page_parent: index == 0
+        }
       end
 
-      ordered_parents << { title: "Home", url: "/" }
+      ordered_parents << {
+        title: "Home",
+        url: "/",
+        is_page_parent: !ordered_parents.any? }
```

Comment:
Oh yeah. I didn't add it because `empty?` is only defined on `Array` and not `Enumerable`, and completely missed the fact that `ordered_parents` is defined as an array below, not passed in as a parameter.

Fixed.